horreum-backend/pom.xml

```diff
@@ -40,6 +41,24 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- commented out until 23.1.1 -->
+<!--        <dependency>
+            <groupId>org.graalvm.polyglot</groupId>
+            <artifactId>polyglot</artifactId>
+            <version>${graalvm.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.graalvm.polyglot</groupId>
+            <artifactId>js-community</artifactId>
+            <version>${graalvm.version}</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>-->
```

Comment:
Can we remove these comments and handle in https://github.com/Hyperfoil/Horreum/pull/1571? otherwise we are going to get a conflict here